Fix: settings releated issues, prepare it for testing (#8416)
deskflow/deskflow: doc/configuration.md

# Gui Config

The search path for settings ar as follows:
 
 - Check Install_path/settings/Deskflow.conf
 - Check if User Settings Path
 - Check if System Settings File

>  - Make a User Setting file

Reviewer: I'm slightly confused about this last check. Perhaps it should be separate to the list but under it, e.g.:

> If no settings file is found then the settings are saved to the user settings path.

>  If no settings are found a new user settings file will be used

Reviewer: Very minor nit: Grammar/phrasing
```suggestion
A new settings file will be created in the user path if no settings file is found.
```

> # Gui Config
> 
> The search path for settings ar as follows:
>  
>  - Check Install_path/settings/Deskflow.conf
>  - Check if User Settings Path
>  - Check if System Settings File
>  - Make a User Setting file
>  
>  The path of the settings file will be used as the base for all other config files

Reviewer: ```suggestion
# GUI Config

The search path for settings is as follows:
 
 - Check <install-path>/settings/Deskflow.conf
 - Check in the user settings path
 - Check in the system settings file
 - Make a user setting file
 
The path of the settings file will be used as the base for all other config files.
```

Author: anything in   < > markdown will eat so i'll quote the path here too
